test/ngMock/angular-mocksSpec.js

```diff
@@ -1106,6 +1116,22 @@ describe('ngMock', function() {
     });
 
 
+    it('should respond with last matched definition', function() {
```

Comment:
Mention `whenOverride` in the description or it is contradicting the one above.

Reply:
Something like "should match against `whenOverride` handlers before normal `when` handlers"?